flaml/nlp/dataset/dataprocess_auto.py

```diff
@@ -178,12 +208,14 @@ def __init__(self):
         )
 
     @classmethod
-    def from_model_and_dataset_name(cls,
-                                    data_raw,
-                                    model_checkpoint_path,
-                                    dataset_name_list: list = None,
-                                    subdataset_name=None,
-                                    **kwargs):
+    def from_model_and_dataset_name(
```

Comment:
Does the tokenizer depend on the task type? If so, task type needs to be an input argument.